ISISScientificComputing/autoreduce
# 602 autoreduction icat instrument mappings

### Summary of work
Replaced the static ICAT_PREFIX_MAP in scripts/manual_operations/manual_submission.py with a dynamic map found by querying ICAT for instrument prefixes.

### How to test your work
- Code review
- No drop in coverage
- Unit tests pass
- Ensure correct ICAT & Autoreduction terminology used
- Ensure that manual submission works for instruments that use a shortened prefix in ISISICAT such as MARI.

Fixes #602
